addons/web/static/tests/views/list_tests.js

```diff
@@ -428,6 +428,286 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('list view: action button in controlPanel basic rendering', async function (assert) {
+        assert.expect(12);
+
+        // LPE FIXME
+        // mockServer doesn't do the right thing (ie python like) for modifiers
```

Comment:
Could you make it works? It would have a large added value for those tests. I think it works in some views, or at some places, no?

Reply:
I guess I could, but my fixme comment was incomplete: there is not any implementation of complex arch processing at all in the mockServer ......

Reply:
I thought [this](https://github.com/odoo/odoo/blob/master/addons/web/static/tests/helpers/mock_server.js#L274) was what you need